image_geometry/package.xml

```diff
@@ -13,17 +13,21 @@
   <url>http://www.ros.org/wiki/image_geometry</url>
 
   <export>
+    <build_type>ament_cmake</build_type>
     <rosdoc config="rosdoc.yaml" />
   </export>
 
-  <buildtool_depend version_gte="0.5.68">catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <build_depend>boost</build_depend>
   <build_depend>opencv3</build_depend>
```

Comment:
can you use "depend" for opencv3? (it is equivalent to build_depend + exec_depend + build_export_depend)